en/tutorials/camera_trigger.md

```diff
@@ -1,86 +1,129 @@
 
-**TODO : needs updating**
+The camera trigger driver supports the following commands.
 
-The camera trigger driver supports several commands -
+### MAV_CMD_DO_TRIGGER_CONTROL
 
-**`MAV_CMD_DO_TRIGGER_CONTROL`** - Accepted in "command controlled" mode (`TRIG_MODE` 1).
+[MAV_CMD_DO_TRIGGER_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_TRIGGER_CONTROL) - Accepted in `TRIG_MODE 1` ("command controlled" mode).
 
 | Command Parameter | Description |
 | -- | -- |
-| Param #1 | Trigger enable/disable (set to 0 for disable, 1 for start) |
-| Param #2 | Trigger cycle time in milliseconds (sets the `TRIG_INTERVAL` parameter. ) |
-| Param #3 | Sequence reset (set to 1 to reset image sequence number, 0 to keep current sequence number) |
+| Param #1 | Trigger enable/disable (set to 0 for enable, 1 for start) |
+| Param #2 | Sequence reset (set to 1 to reset image sequence number, 0 to keep current sequence number) |
+| Param #3 | Trigger pause/restart (set to 0 for pause, 1 for restart). Pausing does not switch off/retract the camera. |
+
 
-**`MAV_CMD_DO_DIGICAM_CONTROL` ** - Accepted in all modes. This is used by the GCS to test-shoot the camera from the user interface. The trigger driver does not yet support all camera control parameters defined by the MAVLink spec.
+### MAV_CMD_DO_DIGICAM_CONTROL
+
+[MAV_CMD_DO_DIGICAM_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONTROL) - Accepted in all modes. This is used by the GCS to test-shoot the camera from the user interface. The trigger driver does not yet support all camera control parameters defined by the MAVLink spec. <!-- deprecated? -->
 
 | Command Parameter | Description |
 | -- | -- |
 | Param #5 | Trigger one-shot command (set to 1 to trigger a single image frame) |
+| Param #7 | Trigger test shot - not logged or forwarded to GCS for geotagging (set to 1 to trigger a test shot) |
+
+
+### MAV_CMD_DO_SET_CAM_TRIGG_DIST
+
+[MAV_CMD_DO_SET_CAM_TRIGG_DIST](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_DIST) - Accepted in "mission controlled" mode (`TRIG_MODE` 4)
+
+This command is auto-generated during missions to trigger the camera based on survey missions from the GCS.
+
+Command Parameter | Description |
+-- | --
+Param #1 | Trigger distance between shots (if `>0`). `0`: triggering paused. `-1`: triggering disabled.
+Param #2 | Trigger activation time in ms - for GPIO mode only (`TRIG_INTERFACE 1`)
+Param #2 | Trigger single shot immediately (if >0)
 
-**`MAV_CMD_DO_SET_CAM_TRIGG_DIST`** - Accepted in "mission controlled" mode (`TRIG_MODE` 4)
 
-This command is autogenerated during missions to trigger the camera based on survey missions from the GCS.
+### MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL
 
-### Testing trigger functionality
+[MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL) - Accepted in "mission controlled" mode (`TRIG_MODE` 4)
```

Comment:
As far as I can tell, this will be used in any interval mode - not just missions. Reason being it just sets the interval but mode not checked. Can you confirm?

Reply:
@ndepal Can you comment on this?